experiment_buddy/utils.py

```diff
@@ -64,7 +64,8 @@ async def __remote_time_logger(elapsed: str):
     elapsed, function_name = re.search(r'([\d.]+).+in (\w+)', elapsed).groups()
 
     async with aiohttp.ClientSession() as session:
-        async with session.get(f'http://static.92.155.21.65.clients.your-server.de/{function_name}/{elapsed}') as response:
+        async with session.get(
+                f'http://static.92.155.21.65.clients.your-server.de/{function_name}/{elapsed}') as response:
```

Comment:
What happened? if it is a voluntary line split, shall we enforce a linter in github actions? so this things do not happen?

Reply:
No, I think it was Pycharm, probably we should